sdk/src/main/java/com/meniga/sdk/MenigaSettings.java

```diff
@@ -39,10 +41,11 @@
 	private final Map<Service, String> specialServiceEndpoints;
 	private final List<Interceptor> interceptors;
 	private final List<Interceptor> networkInterceptors;
+	private final CertificatePinner certificatePinner;
```

Comment:
Where (or how) is the certificatePinner used?

Reply:
The flow looks like this:

```
String hostname = "publicobject.com";
     CertificatePinner certificatePinner = new CertificatePinner.Builder()
         .add(hostname, "sha256/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=")
         .build();
     OkHttpClient client = OkHttpClient.Builder()
         .certificatePinner(certificatePinner)
         .build();

     Request request = new Request.Builder()
         .url("https://" + hostname)
         .build();
     client.newCall(request).execute();
```

Source: https://square.github.io/okhttp/3.x/okhttp/okhttp3/CertificatePinner.html

So in our case we would set it in `MenigaSettings.Builder`:

```
CertificatePinner certificatePinner = new CertificatePinner.Builder()
         .add(hostname, "sha256/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=")
         .build();
menigaSettingsBuilder.certificatePinner(certificatePinner);
```